Gemfile

```diff
@@ -13,7 +20,8 @@ group :development do
 end
 
 group :asciidoc do
-  gem 'asciidoctor'
+  # Asciidoctor 2.0 drops support for Ruby < 2.3.
+  gem 'asciidoctor', (ruby_version?('< 2.3') ? '< 2' : '>= 0')
```

Comment:
`RUBY_VERSION < '2.3'` seems good enough here.

Reply:
Changed according to suggestions.